### DISCUSSION TITLE
BiblioteQ: is Windows 64-bit version possible?

### DISCUSSION THREAD

**Discussion by thiagupillai** [textbrowser/biblioteq#101] [Q&A] [answered]
Biblioteq : Is Win 64 bit version Possible ?

**Comment by textbrowser** [comment]
Hello. I believe the current release should work on W64. Are you asking if a native 64-bit version is possible? I think so. I don't have a Windows 64-bit OS.

**Comment by textbrowser** [comment]
Correction, I have a virtual Windows 64 installation. However, I do not have the proper build environment for a true 64 build.

**Comment by textbrowser** [comment, marked as answer]
Yes, the answer is yes. You'll need a 64-bit machine and Qt 6.2 LTS.

https://en.wikipedia.org/wiki/List_of_platforms_supported_by_Qt#Official_platforms
